devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -107,6 +107,20 @@ def parse(String description) {
 	Map eventMap = zigbee.getEvent(description)
 	Map eventDescMap = zigbee.parseDescriptionAsMap(description)
 
+	if ( device.getDataValue("deviceJoinName") == "eZEX Switch 1") {
+		if (eventMap && eventMap?.name == "switch" && (eventMap?.value == "on" || eventMap?.value == "off")) {
+			if (eventDescMap && eventDescMap?.attrId != "0000") {
+				return [:]
+			}
+		}
+
+		if (!eventMap && eventDescMap) {
+			if (eventDescMap?.clusterId == zigbee.ONOFF_CLUSTER && eventDescMap?.attrId != "0000") {
+				return [:]
+			}
+		}
```

Comment:
I believe the code below:

```
	if (!eventMap && eventDescMap) {
		eventMap = [:]
		if (eventDescMap?.clusterId == zigbee.ONOFF_CLUSTER) {
			eventMap[name] = "switch"
			eventMap[value] = eventDescMap?.value
		}
	}
```

is probably the source of the issue in the first place. You may want to refactor that instead of adding these checks here.

Reply:
@KKlimczukS do you know why you added this code for the Aurora double switch? Do you remember what message was being sent?

Reply:
Okay, then, How about this?
`
	if (eventMap) {
		if (eventDescMap) {
			if (eventDescMap?.attrId != "0000") {
				eventMap = [:]
			}
		}
	} else {
		if (eventDescMap) {
			eventMap = [:]
			if (eventDescMap?.clusterId == zigbee.ONOFF_CLUSTER && eventDescMap?.attrId == "0000") {
				eventMap[name] = "switch"
				eventMap[value] = eventDescMap?.value
			}
		}
	}
`

Reply:
> @KKlimczukS do you know why you added this code for the Aurora double switch? Do you remember what message was being sent?

@greens That must have been quite a long time ago and that's was probably because zigbee.getEvent(description) didn't create the eventMap in a first place.

Reply:
I think you should just be able to remove this code

```
	if (!eventMap && eventDescMap) {
		eventMap = [:]
		if (eventDescMap?.clusterId == zigbee.ONOFF_CLUSTER) {
			eventMap[name] = "switch"
			eventMap[value] = eventDescMap?.value
		}
	}
```

since the device it was added for is no longer on this DTH. Try that and see if it solves the issue.